Rakefile

```diff
@@ -19,6 +19,7 @@
 require "date"
 require "open-uri"
 require "tmpdir"
+require "x"
```

Comment:
Can we make this optional?
This isn't needed on local, right?

Reply:
fix: [d250ee3](https://github.com/groonga/groonga/pull/2294/commits/d250ee38ae3c92d8f40e5ecb80a2c19ecd7e5c4e) Yes, we just want to use it on the release workflow